backend/hct_mis_api/apps/account/schema.py

```diff
@@ -203,7 +203,14 @@ def resolve_user_status_choices(self, info: Any) -> List[Dict[str, Any]]:
         return to_choice_object(USER_STATUS_CHOICES)
 
     def resolve_user_partner_choices(self, info: Any) -> List[Dict[str, Any]]:
-        return to_choice_object(Partner.get_partners_as_choices())
+        business_area_slug = info.context.headers.get("Business-Area")
+        return to_choice_object(
+            list(
+                Partner.objects.exclude(name="Default Empty Partner")
```

Comment:
This ""Default Empty Partner" string should be stored in some global var, it's used few times in our code